sapi/cli/tests/bug60471.phpt

```diff
@@ -0,0 +1,47 @@
+--TEST--
+Bug #60471 Random "Invalid request (unexpected EOF)" using a router script
+--SKIPIF--
+<?php
+include "skipif.inc";
+if (!PHP_DEBUG) die("skip preconnect messages only show in debug mode");
+?>
+--FILE--
+<?php
+include "php_cli_server.inc";
+$stderr_file = __DIR__ . "/bug60471.cli-server.stderr.out";
+$server = php_cli_server_start('echo "Foo";', null, null, null, $stderr_file);
+
+# Partial HTTP request (Unexpected EOF)
+$sock = fsockopen(PHP_CLI_SERVER_HOSTNAME, PHP_CLI_SERVER_PORT);
+fwrite($sock, "HEAD / HTTP/1.0\r\n");
+fclose($sock);
+
+# Mimmik a preconnect
+$sock = fsockopen(PHP_CLI_SERVER_HOSTNAME, PHP_CLI_SERVER_PORT);
+fclose($sock);
+
+# Normal request
+file_get_contents('http://'.PHP_CLI_SERVER_ADDRESS);
+
+php_cli_server_stop($server);
+
+echo file_get_contents($stderr_file);
+unlink($stderr_file);
```

Comment:
I believe this should go into a `--CLEAN--` section: https://qa.php.net/phpt_details.php#clean_section

Reply:
Yeah, I thought about putting that in a `--CLEAN--` section originally and now I can't remember my reasonings for not doing it. I'll add it though :)